src/server/api/controllers/favorite.vessels.controller.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+//POST
+const createFavoriteVessles = async ({ body }) => {
+  const { user_id, vessel_id } = body;
+  const checkUserId = await knex
+    .from("users")
+    .select("*")
+    .where({ id: user_id });
+
+  const checkVesselsId = await knex
+    .from("vessels")
+    .select("*")
+    .where({ id: vessel_id });
+
+  if (checkUserId.length === 0 || checkVesselsId.length === 0) {
+    throw new HttpError(
+      "Bad request",
+      "user_id or vessel_id was not found!",
+      404
+    );
+  }
+
+  return knex("favorite_vessels").insert({ user_id, vessel_id });
+};
+
+//DELETE
+const deleteFavoriteVessels = ({ body }) => {
```

Comment:
Again. Let's keep it singular, since you are only deleting one vessel at a time.

Reply:
Done ;)